Stubs/Xamarin.Forms.Platform.iOS/Xamarin.Forms.Platform.iOS (Forwarders).csproj

```diff
@@ -15,7 +15,7 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
+    <DebugType>portable</DebugType>
```

Comment:
isn't this redundant with the one in Build.props ?

Reply:
Took out all `DebugType` tags for the relevant `csproj` files so that is now all controlled from the `Build.props` for all config types